securedrop/manage.py

```diff
@@ -210,6 +242,48 @@ def listdir_fullpath(d):
     return 0
 
 
+def translate(args):
+    messages_file = os.path.join(args.translations_dir, 'messages.pot')
+
+    if args.extract_update:
+        sh("""
+        set -xe
+
+        mkdir -p {translations_dir}
+
+        pybabel extract \
+        --charset=utf-8 \
+        --mapping={mapping} \
+        --output={messages_file} \
+        --project=SecureDrop \
+        --version={version} \
+        --msgid-bugs-address='securedrop@freedom.press' \
+        --copyright-holder='Freedom of the Press Foundation' \
+        {sources}
+
+        sed -i '/^#, fuzzy$/d' {messages_file}
+
+        test $(ls {translations_dir} | wc -l) = 1 && exit 0
```

Comment:
Add a clarifying comment here too, e.g. `# Continue if there's one file in translations directory as expected`